lib/initConf.js

```diff
@@ -10,6 +10,7 @@ var defaults = {
   LDAP_SEARCH_GROUPS:                   '(member:1.2.840.113556.1.4.1941:={0})',
   LDAP_USER_BY_NAME:                    '(sAMAccountName={0})',
   LDAP_NUMBER_OF_PARALLEL_BINDS:        1,
+  LDAP_HEARTBEAT_SEARCH_QUERY:          '(&(objectclass=user)(|(sAMAccountName=foo)(UserPrincipalName=foo)))',
```

Comment:
We just need a valid query which may or may not return result. This query is also used when setting up connector - https://github.com/auth0/ad-ldap-connector/blob/cbff24a2090901d7d5aeccb5406a32ac13aad8dc/admin/test_config.js#L34

Reply:
`LDAP_HEARTBEAT_SEARCH_QUERY ` config can be overriden in config.json